android/src/main/java/com/github/wuxudong/rncharts/data/DataExtract.java

```diff
@@ -113,7 +113,6 @@ private void drawBitmap(ThemedReactContext context, Canvas c, ReadableMap icon)
                 p.setDither(true);
                 p.setFilterBitmap(true);
                 p.setColorFilter(new PorterDuffColorFilter(color, PorterDuff.Mode.SRC_ATOP));
-                Log.d("React-native-charts-wrapper", "drew bitmap " + color);
```

Comment:
too noisy?  better to comment out if this matches a line in the wuxudong repo?

Reply:
added this line of comment on my own PR here https://github.com/indiemoney/react-native-charts-wrapper/pull/5

Reply:
:ok_hand: